contrib/initramfs/hooks/zfs.in

```diff
@@ -105,4 +105,15 @@ if [ ! -x "$DESTDIR/usr/bin/net" ]; then
     chmod +x "$DESTDIR/usr/bin/net"
 fi
 
+#These are optional files to support tpm2-autounlock
+if [ -x /usr/bin/tpm2_startauthsession ]; then
+    copy_exec /usr/bin/tpm2_nvread
+    copy_exec /usr/bin/tpm2_nvreadlock
+    copy_exec /usr/bin/tpm2_startauthsession
+    copy_exec /usr/bin/tpm2_policypassword
+    copy_exec /usr/bin/tpm2_policypcr
+    copy_modules_dir kernel/drivers/char/tpm
+    copy_exec /usr/lib/x86_64-linux-gnu/libtss2-tcti-device.so.0
```

Comment:
Are the tpm2 tools x86_64 only? (I could see how they might be, though they probably also work on x32 and maybe work in i386?) If not, then the architecture here should probably be detected.